src/controller/java/BUILD.gn

```diff
@@ -215,6 +215,7 @@ android_library("java") {
   data_deps = [ ":jni" ]
 
   sources = [
+    "generated/java/chip/devicecontroller/ClusterIDMapping.java",
     "generated/java/chip/devicecontroller/ClusterReadMapping.java",
```

Comment:
technically   android chip-tool don't depends on these old generated file, right? We just keep them here in case some folks still need them.

"generated/java/chip/devicecontroller/ClusterReadMapping.java",
"generated/java/chip/devicecontroller/ClusterWriteMapping.java",
"src/chip/clusterinfo/ClusterCommandCallback.java",

Reply:
I know that the ".java" file is not currently being created in the out folder.
Do you know anything else?

![image](https://user-images.githubusercontent.com/85541460/233533079-479a91f7-ef72-472a-a27c-6d8361a282c8.png)